.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: core-sdk CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Do we think there is value in running cross OS here on runners such as `macos-latest` or `windows-latest`? They could easily be added to the matrix setup below.

Reply:
I don't think we have anything that is OS-specific in the SDK so I think I would just leave it as you have it unless we add something like that in the future or run into platform-specific issues at some point.